Example/Authenticator/View Controllers/Connection/ConnectionsViewController.swift

```diff
@@ -248,4 +249,18 @@ extension ConnectionsViewController: ConnectionCellEventsDelegate {
     func consentsPressed(id: String) {
         viewModel.consentsPressed(connectionId: id)
     }
+    
+    private func checkInternetConnection(id: String) {
```

Comment:
move this function in `ViewControllerExtensions` and make it public, add there a new parameter confirmAction as completion block.

`func checkInternetConnection(id: String, comfirmAction: @escaping () -> ())`

Make sure you use this method also in other controller, where we need to check internet connection.

Reply:
we discussed the changes